setup.rst

```diff
@@ -106,24 +106,20 @@ Basing your Project on a Specific Symfony Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In case your project needs to be based on a specific Symfony version, use the
-optional second argument of the ``new`` command:
+``version`` option of the ``new`` command:
 
 .. code-block:: terminal
 
     # use the most recent version in any Symfony branch
-    $ symfony new my_project_name 2.8
-    $ symfony new my_project_name 3.1
+    $ symfony new --version=3.3 my_project_name
+    $ symfony new --version=3.4 my_project_name
 
     # use a specific Symfony version
-    $ symfony new my_project_name 2.8.3
```

Comment:
Any version lower to 3.3 does not work because `symfony/skeleton` did not exist (`Could not find package symfony/skeleton with version 2.8.3.`).